--- action.php.orig     2023-02-08 00:05:10.758010050 -0500
+++ action.php  2023-02-08 00:07:57.716003720 -0500
@@ -22,8 +22,9 @@
         $oauth = $this->getOAuthService();
         $data = array();

-        $url = $this->getConf('userurl');
-        $raw = $oauth->request($url);
+        $access_token =  $oauth->getStorage()->retrieveAccessToken($oauth->service());
+        $url = $this->getConf('userurl') ."/". $access_token->getAccessToken();
+        $raw = $oauth->request($url,'GET',null,array("Accept" => "application/json"));

         if (!$raw) throw new OAuthException('Failed to fetch data from userurl');
         $result = json_decode($raw, true);